src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.js

```diff
@@ -220,16 +222,42 @@ function ComposerWithSuggestions({
                     debouncedUpdateFrequentlyUsedEmojis();
                 }
             }
-            const newCommentConverted = convertToLTRForComposer(newComment);
+
+            let newCommentConverted = convertToLTRForComposer(newComment);
+
+            const moveCursorToEndOfLine = (commentLength) => {
+                setSelection({
+                    start: commentLength + 1,
+                    end: commentLength + 1,
+                });
+            };
+
+            if (firstChange && commentRef.current === '' && !['@'].includes(newComment)) {
```

Comment:
Please add code comments to explain _why_ all this logic is here.

Reply:
Got it, done!